### PR TITLE
wrote test for leaderboard/get_static_info?addr=${addr}&duration=${duration}

### LINKED ISSUE #612: test: `/leaderboard/get_static_info?addr=${addr}&duration=${duration}`
read description here - https://github.com/starknet-id/starknet.quest/issues/602#issue-2255907921

### PR DESCRIPTION
**Pull request by Ugo-X**
#612 

# Pull Request type

- Testing

Resolves: #612

### CODE REVIEW

File: tests/services/leaderboard.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    expect(fetch).toHaveBeenCalledWith(
+      `${API_URL}/leaderboard/get_static_info?addr=${params.addr}&duration=${params.duration}`
+    );
+    expect(response).toEqual(mockResponse);
+  });
+
+  it("handles errors gracefully", async () => {
+    const params = { addr: "exampleAddr", duration: "exampleDuration" };
+
+    const mockResponse = "Leaderboard positions not found";
```
Comment (ayushtom) [inline tests/services/leaderboard.test.js]: is this the response you get when there is no position found?
i think it should be something like - 'Error querying ranks'

Comment (Ugo-X) [reply tests/services/leaderboard.test.js]: Should I edit the response?

Comment (Ugo-X) [reply tests/services/leaderboard.test.js]: i just modified the response. opening a new pull request

Comment (Ugo-X) [reply tests/services/leaderboard.test.js]: > is this the response you get when there is no position found? i think it should be something like - 'Error querying ranks'
I’ve edited the response message

**Pull request closed without merging**